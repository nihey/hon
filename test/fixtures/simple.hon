# Once upon a time, at an abandoned amusement park
echo "I'm here to see Joseph Clarence"

python
    print "Never heard of him.."

echo "Mr. Swing King"

python
    print "Oh, sorry ass dickhead... What the fuck they call you?"
    # I'm still not sure about this baldie
    relaxed = True

echo "Names are for friends...so I don't need one."

node
    console.log('They Played Us Like A Damn Fiddle!')
    var state = 'enraged'
    console.log('[STATE=' + state + '] Give it back that was ours!')
    # Damn.. Boss...
