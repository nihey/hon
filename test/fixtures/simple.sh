#!/bin/bash

echo "I'm here to see Joseph Clarence"
echo "print \"Never heard of him..\"" | python
echo "Mr. Swing King"
echo "print \"Oh, sorry ass dickhead... What the fuck they call you?\";relaxed = True" | python
echo "Names are for friends...so I don't need one."
echo "console.log('They Played Us Like A Damn Fiddle!');var state = 'enraged';console.log('[STATE=' + state + '] Give it back that was ours!')" | node