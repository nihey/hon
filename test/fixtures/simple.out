I'm here to see Joseph Clarence
Never heard of him..
Mr. Swing King
Oh, sorry ass dickhead... What the fuck they call you?
Names are for friends...so I don't need one.
They Played Us Like A Damn Fiddle!
[STATE=enraged] Give it back that was ours!
